gnome-image-installer/pages/confirm/gis-confirm-page.ui

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="GisConfirmPage" parent="GisPage">
+    <child>
+      <object class="GtkBox" id="page">
+        <property name="visible">True</property>
+        <property name="can_focus">False</property>
+        <property name="valign">center</property>
+        <property name="margin_left">64</property>
+        <property name="margin_right">64</property>
+        <property name="orientation">vertical</property>
+        <property name="spacing">16</property>
+        <child>
+          <object class="GtkGrid" id="computer_grid">
+            <property name="visible">True</property>
+            <property name="can_focus">False</property>
+            <property name="halign">center</property>
+            <property name="hexpand">True</property>
+            <property name="row_spacing">5</property>
+            <property name="column_spacing">12</property>
+            <child>
+              <object class="GtkLabel" id="vendor_caption">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="halign">end</property>
+                <property name="hexpand">False</property>
+                <property name="label" translatable="yes">Computer Vendor</property>
+                <property name="xalign">1</property>
+                <accessibility>
+                  <relation type="label-for" target="vendor_label"/>
+                </accessibility>
+                <style>
+                  <class name="dim-label"/>
+                </style>
+              </object>
+              <packing>
+                <property name="left_attach">0</property>
+                <property name="top_attach">0</property>
+              </packing>
+            </child>
+            <child>
+              <object class="GtkLabel" id="vendor_label">
+                <property name="visible">True</property>
+                <property name="can_focus">False</property>
+                <property name="halign">start</property>
+                <property name="hexpand">True</property>
+                <property name="label">To Be Filled By OEM</property>
```

Comment:
Should this be translatable? (‘No’ is a fine answer.)

Reply:
No, it's an in-joke placeholder that will always be replaced at runtime.

Reply:
The best.